shelley/design-spec/delegation_design_spec.tex

```diff
@@ -1248,15 +1179,14 @@ \subsubsection{Stake Pool Registration Certificates}
 
 \begin{itemize}
 \item
-  The public staking key hash \(\mathcal{H}(vks_\text{pool})\) of the
-  pool.
+  The public key hash \(\mathcal{H}(vk_\text{pool})\) of the pool.
 \item
   The epoch number, starting from which the stake pool will cease to
   operate.
 \end{itemize}
 
-It must be signed by the staking key \(sks_\text{pool}\) of the pool
-(particularly, it need not be signed by any of the pool owners).
+It must be signed by the pool key \(sk_\text{pool}\) (particularly, it need not
+be signed by any of the pool owners).
```

Comment:
Is pool key defined?  This is one of the points of potential confusion that the pool operators have spotted

Reply:
I think it is, but I'll make it more explicit to be sure.